README.md

```diff
@@ -26,57 +28,72 @@ After running the command, there will be a few prompts:
     -  [Gitpod](https://www.gitpod.io)
   Dev Containers also have integration with [Wokwi simulator](https://wokwi.com/) and allow flashing from the container using [web flash](https://github.com/bjoernQ/esp-web-flash-server).
 
-![CI](https://github.com/esp-rs/esp-idf-template/actions/workflows/ci.yml/badge.svg)
-
 ## Prerequisites
 
-### Install Rustup
-
-If you don't have `rustup` installed yet, follow the instructions on the [rustup.rs site](https://rustup.rs)
-
-### Install Rust & Clang - for Xtensa MCUs (ESP32, ESP32-S2 and ESP32-S3)
-
-- Install the [Rust Espressif compiler toolchain and the Espressif LLVM Clang toolchain](https://github.com/esp-rs/rust-build)
-- This is necessary, because support for the Xtensa architecture (ESP32 / ESP32-S2 / ESP32-S3) is not upstreamed in LLVM yet
-- Make sure that you DON'T have a system Clang installed as well, because even if you have the Espressif one first on your `$PATH`, Bindgen will still pick the system one
-  - A workaround that does not require uninstalling the system Clang is to do `export LIBCLANG_PATH=<path to the Espressif Clang lib directory>` prior to continuing the build process
-
-### Install Rust & Clang - for RiscV32 MCUs (ESP32-C3)
+### Install Python3
 
-- You **can** target the ESP32-C3 with the Espressif toolchains just fine, but this MCU is also supported by the stock compilers
-- So alternatively - just use the stock nightly Rust compiler, and a recent, stock Clang (as in Clang 11+)
+You need a Python 3.7 or later installed on your machine. Install it from the package distro of your OS, or download and install it [from the official Python site](https://www.python.org/downloads/).
```

Comment:
True, I just added the link to the book where we explain this, but it would be better to have it explicit. 
[35df5f5](https://github.com/esp-rs/esp-idf-template/pull/69/commits/35df5f5fdcd74eadbfc9e03d5131a45d8e333a80) should fix this

Reply:
Unless there are good reasons not to, how about also restoring:
- The Rustup installation guidelines; or else how are they going to call `rustup toolchain`?
- The link to the Clang site.

Reply:
You are right! Restored in [304ec34](https://github.com/esp-rs/esp-idf-template/pull/69/commits/304ec341465779ccbe1a774f69c417225e8343ae)

Thanks for the feedback